tool/tool.py

```diff
@@ -19,6 +35,12 @@ class Tool:
     # These commands will be executed in the order they are provided.
     install_commands: List[str]
 
+    # Mention what are the environment variables that need to be copied from your current environment
```

Comment:
Won't there any any `usage_instructions`. I assume it will be part of description